pkg/autoscaler/scaling/autoscaler.go

```diff
@@ -54,7 +58,7 @@ type autoscaler struct {
 	// specMux guards the current DeciderSpec and the PodCounter.
```

Comment:
Please change this comment and move podCount to the first section.

Reply:
ok. Done.